CONTRIBUTING.md

```diff
@@ -132,8 +137,37 @@ meantime.
 
 It can be helpful to go through older bug reports and make sure that they are
 still valid. Load up an older issue, double check that it's still true, and
-leave a comment letting us know if it is or is not. The [least recently updated sort][lru] is good for finding issues like this.
+leave a comment letting us know if it is or is not. The [least recently
+updated sort][lru] is good for finding issues like this.
+
+Contributors with sufficient permissions on the Rust repo can help by adding
+labels to triage issues:
+
+* Yellow, **A**-prefixed labels primarily relate to what part of the project
+  an issue is **about**. 
+
+* Magenta, **B**-prefixed labels identify bugs which **belong** elsewhere. 
+
+* Green, **E**-prefixed labels flag **easy** or introductory bugs. 
+
+* Red, **I**-prefixed labels warn of serious **issues** with the language
+  itself, such as regressions and slowdowns.
+
+* Orange, **P**-prefixed labels indicate a bug's **priority**. 
+
+* Blue, **T**-prefixed bugs denote where in the **toolchain** the issue fits.
+
+* Dark blue, **beta-** and **doc-backport** tags have never been applied to
+  any issues. 
```

Comment:
Ah these are actually somewhat transient tags in the sense that they're only applied for backporting changes to the beta branches, so there's rarely any *open* issues/PRs with these tags, but there's a number of closed ones. For example there's a few `beta-nominated` PRs that are open, and the `beta-accepted` tag is mostly only found on closed PRs.